tools/depmat.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+import sys
+from getopt import gnu_getopt, GetoptError
+from re import compile as Regex
+from collections import namedtuple
+
+from libgcopdd import Event, UserError, args_to_events, n_args_to_n_events, open_dump_entry, create_temp
+import phasestack
+
+INTEGER_PATTERN = Regex(r'\d+')
+DepMat = namedtuple('DepMat', 'size, mapping, data')
```

Comment:
What does the mapping mean? That's not clear to me.

Reply:
(a few hours later)

I have accidentally (:grin:) explained what mapping is in review comment at `phasestack.get_mapping()` ([phasestack.py:28](https://github.com/vakabus/gcopdd/pull/15/files/692830ba722441b3eca3c6cb4f633a7afba24c4c#diff-efc0b25d9a24e25b61e3e302a75ca658R28)).